spark_pipeline_framework/transformers/send_automapper_to_fhir/v1/automapper_to_fhir_transformer.py

```diff
@@ -76,6 +79,12 @@ def __init__(
         :param run_synchronously: (Optional) Run on the Spark master to make debugging easier on dev machines
         :param additional_request_headers: (Optional) Additional request headers to use
                                             (Eg: {"Accept-Charset": "utf-8"})
+        :param sort_data: (Optional) Whether to sort the data. Format - {
+                            views: [List of views],
+                            column_for_sorting: "columnName to be used for sorting",
+                            drop_column: bool (Whether to drop the column used for sorting),
+                            partition_by_column_name: (str) Name of the column that will be used to repartition df
+                        }
```

Comment:
column_for_sorting, drop_column, etc should be per view and not the same for all views.